Processing for iOS/ProjectSelectionTableViewController.swift

```diff
@@ -164,25 +164,22 @@ class ProjectSelectionTableViewController: UITableViewController, UIViewControll
     }
     
     @IBAction func createNewProject(_ sender: Any) {
-        let alertView = UIAlertView(title: "New Processing Project", message: "", delegate: self, cancelButtonTitle: "Cancel", otherButtonTitles: "Create")
-        alertView.alertViewStyle = .plainTextInput
-        alertView.show()
+        showCreateAlert(title: "New Processing Project", name: "")
     }
     
-    func alertView(_ alertView: UIAlertView, clickedButtonAt buttonIndex: Int) {
-        let textField = alertView.textField(at: 0)
-        textField?.resignFirstResponder()
-        
-        if buttonIndex == 1 {
-            if let fileName = textField?.text {
+    func showCreateAlert(title: String, name: String) {
+        let alertController = UIAlertController(title: title, message: "", preferredStyle: .alert)
+        alertController.addAction(UIAlertAction(title: "Create", style: .default, handler: { (action) in
+            NSLog("SAVE PUSHED")
+            if let fileName = alertController.textFields?[0].text {
                 let letters = NSMutableCharacterSet.letters as! NSMutableCharacterSet
                 letters.addCharacters(in: "-_1234567890")
                 
                 var message = ""
                 var suggestedName = fileName
                 if fileName == "" {
                     message = "Name should be at least one character"
-                } else if nameAlreadyExists(name: fileName) {
+                } else if self.nameAlreadyExists(name: fileName) {
                     message = "File with name '\(fileName)' already exists. Please choose another name or delete the exiting one first."
                 } else if fileName.contains(" ") {
                     message = "File name should not contain spaces."
```

Comment:
I guess this needs to be the case because Processing?

Reply:
The reason why the name should not contain spaces? Yeah, the reason was to not c confuse the folder structure of Processing.